README.md

```diff
@@ -84,6 +84,8 @@ There are no additional requirements for CPU TorchDynamo. CPU TorchDynamo is inc
 
 Build PyTorch from source: https://github.com/pytorch/pytorch#from-source, which has TorchDynamo included.
 
+To install GPU TorchDynamo dependencies, run `make triton`.
```

Comment:
Maybe we should have this. Otherwise some users might think to look in openai repo first

```
cd pytorch
make triton
```